prism-languages.php

```diff
@@ -62,5 +63,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 	 *
 	 * @param string $languages Array of languages supported
 	 */
+	
```

Comment:
:no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).